### DISCUSSION TITLE
Persist Projected Texture

### DISCUSSION THREAD

**Discussion by stripuramallu3** [marcofugaro/three-projected-material#51] [Q&A]
Hi,

I came across this library when trying to project a material onto my mesh and it works great!
I was wondering if anyone has insight on how to persist the projected material. 

For example:
- Project material
- Rotate camera
- Project new material

In the above example I'd like both projections to persist on the mesh.

**Comment by marcofugaro** [comment]
You can use multiple ProjectedMaterials like showed in the [multiple projections example](https://marcofugaro.github.io/three-projected-material/multiple-projections).

**Reply by stripuramallu3** [reply]
Thank you! I think this will work for what I'm trying to do
